src/gcp/cloudfunctions.ts

```diff
@@ -579,44 +579,47 @@ export function functionFromSpec(
  * This API exists outside the GCF namespace because GCF returns an Operation<CloudFunction>
  * and code may have to call this method explicitly.
  */
-export function endpointFromFunction(gcfFunction: CloudFunction): backend.FunctionSpec {
+export function endpointFromFunction(gcfFunction: CloudFunction): backend.Endpoint {
   const [, project, , region, , id] = gcfFunction.name.split("/");
-  let trigger: backend.EventTrigger | backend.HttpsTrigger;
+  let trigger: backend.Triggered;
   let uri: string | undefined;
   if (gcfFunction.httpsTrigger) {
+    trigger = { httpsTrigger: {} };
+    uri = gcfFunction.httpsTrigger.url;
+  } else if (gcfFunction.labels?.["deployment-scheduled"]) {
     trigger = {
-      // Note: default (empty) value intentionally means true
-      allowInsecure: gcfFunction.httpsTrigger.securityLevel !== "SECURE_ALWAYS",
+      scheduleTrigger: {},
```

Comment:
q: why do we not copy over attributes like `schedule`, `timeZone`, and `retryConfig` here?

Reply:
We would have to make a second API request to Cloud Scheduler to fill in these details and historically we have not done this.